addStudy.py

```diff
@@ -0,0 +1,38 @@
+import os
```

Comment:
Be sure to include the proper interpreter directive at the top so the script can be run from bash.